.clang-format

```
@@ -54,14 +55,21 @@ FixNamespaceComments: true
 ForEachMacros:
   - forever # avoids { wrapped to next line
   - foreach
-  - Q_FOREACH
   - BOOST_FOREACH
 IncludeCategories:
   - Regex:           '^<Q.*'
+    Priority:        300
+  - Regex:           '^<qgs.*'
     Priority:        200
-IncludeIsMainRegex: '(test_?)?$'
+  - Regex:           '<.*'
+    Priority:        400
+  - Regex:           '^".*'
+    Priority:        100
+  - Regex:           '.*'
+    Priority:        1
+IncludeIsMainRegex: false
+IncludeBlocks: Regroup
 IndentCaseLabels: true
-IndentPPDirectives: false
 IndentWidth: 2
```

```suggestion
IndentWidth: 4
```

looking at the issue here: https://bugs.llvm.org/show_bug.cgi?id=19056
I think we were better using a 4 indent, so there would be no modification of the indent in the headers (by keeping the access modifiers. offset to -2)

---

Won't that have an immense impact on all code in cpp files?
What we'd want is only the `class` and `struct` set to 4.

---

indeed